errors.go

```diff
@@ -111,7 +111,7 @@ var (
 	}
 	ErrInsufficientEntropy = &RFC6749Error{
 		Name:        errInsufficientEntropyName,
-		Description: fmt.Sprintf("The request used a security parameter (e.g., anti-replay, anti-csrf) with insufficient entropy (minimum of %d characters)", MinParameterEntropy),
+		Description: "The request used a security parameter (e.g., anti-replay, anti-csrf) with insufficient entropy",
```

Comment:
It is not necessary to have the minimal length here because we pass a similar message (with minimal length) as a hint as well.